eng/pipelines/templates/steps/configure-proxy.yml

```diff
@@ -0,0 +1,18 @@
+parameters:
```

Comment:
There is a pretty excellent azure devops feature that I just was notified of. I'm going to see if I can use it.

Reply:
@scbedd do I still need this file or can I rely on the files in `eng/common/testproxy`?

Reply:
You should be able to delete this file now. I'm actually a bit disconcerted that your current setting of the PROXY_CERT variable isn't working properly.